src/Microsoft.ML.CodeGen/Templates/Console/Annotation.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
What's the purpose of this empty file?

Reply:
Oh it's generated by T4... I will remove that